BREAKING.md

```diff
@@ -5,9 +5,9 @@ This is a comprehensive list of the breaking changes introduced in the major ver
 ## Versions
 
 - [Version 6.x](#version-6x)
-- [Version 5.x](#version-5x)
-- [Version 4.x](#version-4x)
-- [Legacy](#legacy)
+- [Version 5.x](./BREAKING_ARCHIVE/v5.md)
+- [Version 4.x](./BREAKING_ARCHIVE/v4.md)
+- [Legacy](./BREAKING_ARCHIVE/legacy.md)
```

Comment:
Currently `legacy.md` just has a link to another file; could we link directly to that one instead and skip the middleman?

Reply:
Works for me!